src/test/ui/type-alias-enum-variants/enum-variant-generic-args-pass.rs

```diff
@@ -1,17 +1,23 @@
 // run-pass
 
-#![feature(type_alias_enum_variants)]
+// Check that resolving, in the value namespace, to an `enum` variant
+// through a type alias is well behaved in the presence of generics.
+// We check for situations with:
+// 1. a generic type `Alias<T>`, we can type-apply `Alias` when referring to a variant.
+// 2. a monotype `AliasFixed` of generic `Enum<T>`, we can refer to variants
+//    and the type-application of `T` in `AliasFixed` is kept.
 
 #![allow(irrefutable_let_patterns)]
 
 #[allow(dead_code)]
```

Comment:
Lints from the `unused` group are `allow`ed by default in UI tests.

Reply:
Fixed in https://github.com/rust-lang/rust/pull/61682/commits/57e6869a6be56cd75e382deb313fb5f4d1bb1fda.